CodeStarterCamp_Week3/main.swift

```diff
@@ -8,5 +8,51 @@
 
 import Foundation
 
-print("Hello, World!")
+enum Coffee {
+    case americano
+    case latte
+    case espresso
+    case fruitSmoothie
+}
 
+struct Person {
+    var name: String
+    var money: Int
+    
+    mutating func buySomething(somethingPrice: Int) {
+        if self.money >= somethingPrice {
```

Comment:
오- 예외처리 좋습니다 👍 
추가 코멘트 살짝 드리자면
```suggestion
        let hasEnoughMoney = self.money >= somethingPrice
        if hasEnoughMoney {
```
요렇게 표현해줄 수도 있을 것 같아요 (이건 조건이 간단해서 취향의 영역일 것 같습니다 😁)

Reply:
아하! if 뒤에는 아무래도 짧고 명확하게 들어가는 게 낫군요! 상수로 선언해서 수정하겠습니다!!